src/attributes.jl

```diff
@@ -721,14 +721,14 @@ The optimizer stopped because of some user-defined limit.
 * `NodeLimit`: a branch-and-bound algorithm stopped because it explored a
   maximum number of nodes in the branch-and-bound tree.
 * `SolutionLimit`: the algorithm stopped because it found the required number of
-  solutions. This is often used in MIP's to get the solver to return the first
+  solutions. This is often used in MIPs to get the solver to return the first
   feasible solution it encounters.
 * `MemoryLimit`: the algorithm stopped because it ran out of memory.
 * `ObjectiveLimit`: the algorthm stopped because it found a solution better than
   a minimum limit set by the user.
 * `NormLimit`: the algorithm stopped because of a criteria involving a norm.
-  This is typically used to terminate an algorithm once successive iterations
-  result in a solution that is within some distance of the previous solution.
+  This is typically used to indicate that an algorithm stopped after
+  encountering a sequence of diverging iterations.
```

Comment:
Diverging sequence is maybe too specific. The limit means that the norm of an iterate became too large.